> 
> 
> @bcoles Sorry for extended delay. Added the code you suggested, though, it's quite unsatisfying as a check.

Agree, but better than nothing.

Looks like the module was chmoded executable. Also, there's a rogue space at EOL on line 77.

```
[*] Running msftidy.rb in ./.git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/dev/msftidy.rb ---
modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb - [ERROR] Module should not be executable (+x)
modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb:77 - [WARNING] Spaces at EOL
```

--- a/modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb
+++ b/modules/exploits/windows/local/ms16_032_secondary_logon_handle_privesc.rb
@@ -74,7 +74,7 @@ class MetasploitModule < Msf::Exploit::Local
       # Non-Windows systems are definitely not affected.
       return Exploit::CheckCode::Safe
     end
-    
+
     res = psh_exec 'if($([System.Environment]::ProcessorCount) -gt 1) { echo("true") }'
     unless res.include? 'true'
       vprint_error 'Target system has an insufficient number of processor cores'

